features/admin.server-configurations.v1/forms/login-attempt-security-form.tsx

```diff
@@ -472,10 +492,28 @@ export const LoginAttemptSecurityConfigurationFrom: FunctionComponent<
                             "loginAttemptSecurity.form.fields.maxFailedAttempts.hint")
                     }
                 </Hint>
+                <Heading as="h5">
+                    { t("extensions:manage.serverConfigurations.accountSecurity." +
+                            "loginAttemptSecurity.accountLockDurationHeading") as ReactNode }
```

Comment:
Is casting to ReactNode necessary here?

Reply:
Resolved by https://github.com/wso2/identity-apps/pull/6997/commits/6685823eccbff5a04b0ec7abe07b44b2dfbe4910